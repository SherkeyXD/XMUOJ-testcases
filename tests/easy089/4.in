8 8 3 7 14 19 18 4 16 9 12 10 1 2 14 15 5 3 
2 5 13 6 10 4 14 9 8 
8 5 5 7 12 16 10 1 4 14 9 16 19 12 18 
8 2 8 5 1 10 6 17 13 12 18 2 
