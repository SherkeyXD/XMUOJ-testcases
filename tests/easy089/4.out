4 7 9 16 18 19 
6 13 
1 4 5 7 10 14 
1 5 6 8 10 12 13 17 
